ui/cryptomaterial/segmented_control.go

```diff
@@ -105,7 +109,13 @@ func (sc *SegmentedControl) Layout(gtx C, body func(gtx C) D, isMobileView ...bo
 				})
 			}),
 		)
-	}, true)
+	}
+
+	if sc.disableUniformPadding {
+		return widget(gtx)
+	}
+
+	return UniformPadding(gtx, widget, sc.isMobileView)
```

Comment:
i think this would do more justice if you do this;
it makes the code easier to read an understand

```suggestion
	if sc.isMobileView {
		return UniformPadding(gtx, widget, sc.isMobileView)
	}

        return widget(gtx)

```

Reply:
It is not related to mobile view, it is related about margin on segmented control because we have 2 segmented control on one page